moveit_core/CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog for package moveit_core
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+2.2.3 (2022-01-07)
+------------------
+* Fix CHOMP motion planner build on Windows (`#890 <https://github.com/ros-planning/moveit2/issues/890>`_) (`#936 <https://github.com/ros-planning/moveit2/issues/936>`_)
+  Co-authored-by: Akash <Ace314159@users.noreply.github.com>
+* Fix boost linking errors (`#900 <https://github.com/ros-planning/moveit2/issues/900>`_) (`#920 <https://github.com/ros-planning/moveit2/issues/920>`_)
+  Co-authored-by: Vatan Aksoy Tezer <vatan@picknik.ai>
+* Contributors: mergify[bot]
```

Comment:
Probably nit-picky, but IMO Contributors should really list the authors (also co-authors) and not Mergify. Also, the inline co-authors should be dropped.

Reply:
renaming it to ament_generate_changelog would be a great first step ;)

Reply:
I manually fixed these as there was only 4 of them.

Reply:
thank you!